book-service-rest/src/test/java/com/example/Library/security/mvnJwtTokenProviderTests.java

```diff
@@ -7,7 +7,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class JwtTokenProviderTests {
+public class mvnJwtTokenProviderTests {
```

Comment:
no 'mvn' prefix needed for test names, test name should begin with the name of class you are testing and end with 'Test' or 'Tests' suffix